burnman/planet.py

```diff
@@ -0,0 +1,113 @@
+from __future__ import print_function
+# This file is part of BurnMan - a thermoelastic and thermodynamic toolkit for the Earth and Planetary Sciences
+# Copyright (C) 2012 - 2015 by the BurnMan team, released under the GNU
+# GPL v2 or later.
+
+import numpy as np
+from scipy.integrate import odeint
+from scipy.integrate import quad
+from scipy.interpolate import UnivariateSpline
+import burnman
+from burnman import minerals as minerals
+
+G = 6.67e-11
+
+
+class Planet(object):
+    def __init__(self, radii, temperatures, compositions):
+
+        self.radii = radii
+        self.temperatures = temperatures
+        self.pressures = np.linspace(350.0e9, 0.0, len(radii)) # initial guess at pressure profile
+        self.compositions = compositions
+        #creates the geotherm, starting with just 300 K isothermal
+        self.averaging_scheme = burnman.averaging_schemes.VoigtReussHill()
+
+    def generate_profiles( self, radii, n_iterations ):
```

Comment:
Is there a reason ``generate_profiles`` is not automatically called when ``Planet`` is constructed?